include/ocpp/common/cistring.hpp

```diff
@@ -23,6 +23,12 @@ template <size_t L> class CiString : public String<L> {
     CiString(const char* data) : String<L>(data) {
     }
 
+    CiString(json data) : String<L>(data.get<std::string>()) {
+    }
+
+    template <size_t LL> CiString(const CiString<LL>& data) : String<L>(data.get()) {
```

Comment:
So this allows us to assign strings of a different length to this type, is that intended?

Reply:
That's really not what we want, I removed that again